presto-main/src/main/java/com/facebook/presto/server/ResourceGroupInfo.java

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
Is this file created by `git mv`? Why it's not showing diff but show as a new file?

Reply:
I think it determined that the diff was large enough that it should be a new file rather than a move.